src/SFA.DAS.AssessorService.Application.Api.External/Startup.cs

```diff
@@ -31,6 +32,7 @@ namespace SFA.DAS.AssessorService.Application.Api.External
 {
     public class Startup
     {
+        private readonly IConfiguration _config;
         private readonly IWebHostEnvironment _env;
         private readonly ILogger<Startup> _logger;
         private const string SERVICE_NAME = "SFA.DAS.AssessorService.ExternalApi";
```

Comment:
Are the SERVICE_NAME and Version used for anything now that the new config is being used?

Reply:
Nope, but I did forget to add them to the appsettings.